tests/testthat/test-get_delay_estimate.R

```diff
@@ -49,11 +42,26 @@ test_that("get_delay_estimate handles partially complete reporting triangles", {
   # Test 8: Test that a partial triangle works correctly
   partial_triangle <- matrix(
     c(
-      80, 50, 25, 10,
-      100, 50, 30, 20,
-      90, 45, 25, 20,
-      80, 40, 15, NA,
-      70, 30, NA, NA
+      80,
```

Comment:
Any particular reason for this syntax? I find it much easier to read if it looks like the matrix you're making

Reply:
its the air formatter we said we should use. I don't feel strongly about it

Reply:
Looks nice otherwise, I just prefer these to be consolidated

Reply:
I have turned off AIR